convert-hf-to-gguf.py

```diff
@@ -192,7 +192,7 @@ def from_model_architecture(model_architecture):
             return RefactModel
         if model_architecture == "PersimmonForCausalLM":
             return PersimmonModel
-        if model_architecture in ("StableLMEpochForCausalLM", "LlavaStableLMEpochForCausalLM"):
+        if model_architecture in ("StableLmForCausalLM", "LlavaStableLMEpochForCausalLM"):
```

Comment:
Should we also keep the ability to convert from older checkouts of the StableLM model files?
I'm pretty sure some finetunes will forget to keep up with this change. (e.g. right now Nous-Capybara-3B-V1.9 still has the older `config.json`, among others).
```suggestion
        if model_architecture in ("StableLmForCausalLM", "StableLMEpochForCausalLM", "LlavaStableLMEpochForCausalLM"):
```

Reply:
I thought abt it for a moment, but for safety let's add it. However, in latest transformers version StableLMEpochForCausalLM doesn't exist anymore, so they'll have to update their config.json anyway.

Reply:
StableLMEpochForCausalLM was never part of transformers AFAIK, it's the pre-transformers custom model class defined in modeling_stablelm_epoch.py.